CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# Unreleased
+
+- add a Creative Commons sample EDX course
```

Comment:
Unclear. This is the scraper changelog. We're not adding a course, we're adding a sample command to retrieve one, in the README.